### DISCUSSION TITLE
How can I revert a commit in Git?

### DISCUSSION THREAD

**Discussion by freezeukees** [community/community#129241] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How can I revert a commit in Git??

**Comment by ruchiralkm** [comment]
To revert a commit in Git, you can use the 'git revert' command followed by the commit hash you want to undo. This command creates a new commit that undoes the changes made by the specified commit, effectively reverting your codebase to its previous state.
